app/assets/javascripts/application.js

```diff
@@ -56,8 +60,17 @@ $(document).on('click', 'a.js-sync', function(e) {
 
 $(document).on('click', 'tr.notification', function() {
   Octobox.markRowCurrent($(this))
-})
+});
 
 $(document).on('click', '.toggle-star', function() {
   Octobox.toggleStarClick($(this))
-})
+});
+
+$(document).on('click', '.subject-link', function () {
+  history.pushState({thread: $(this).attr('href')}, 'Octobox', $(this).attr('href'))
+  $.get($(this).attr('href'), function(data){
+    $('#thread').html(data)
+  });
+  Octobox.openThread();
+  return false;
```

Comment:
We should probably move this to octobox.js

Reply:
moving this (wrapping a pushLink function into octobox.js with this logic inside and passing $this to it) resulted in a strange xhr deprecation warning and a refusal to open the link...

Reply:
Fixed in https://github.com/octobox/octobox/pull/1169/commits/5c26b637afe44d3401e79f9fd8571ed8e64da5d2